src/main/java/com/codahale/metrics/newrelic/NewRelicReporterFactory.java

```diff
@@ -50,12 +49,30 @@ else if (entry.getValue() instanceof Boolean)
           attributes.put(entry.getKey(), (Boolean) entry.getValue());
       }
     }
-    return NewRelicReporterBuilder.forRegistry(registry, metricBatchSender.build())
+    MetricBatchSender metricBatchSender = MetricBatchSender.create(config.build());
+    return NewRelicReporterBuilder.forRegistry(registry, metricBatchSender)
         .durationUnit(getDurationUnit())
         .rateUnit(getRateUnit())
         .filter(getFilter())
         .commonAttributes(attributes)
         .disabledMetricAttributes(disabledMetricAttributes)
         .build();
   }
+
+  private SenderConfiguration.SenderConfigurationBuilder configureEndpoint(
+      SenderConfiguration.SenderConfigurationBuilder config) {
+    if (overrideUri == null) {
+      return config;
+    }
+    final URI uri = URI.create(overrideUri);
+    try {
+      String path = uri.getPath();
+      if (path == null || path.isEmpty()) {
```

Comment:
I don't think this is wrong, but it's definitely a change in the existing behavior, that we should be sure to document when we release this new version.

Reply:
Why is it a change?  There used to just be a single API before whose documentation says that the path of whatever URI you passed it would be ignored.  We now have two APIs, one with full URI including path, and one that's just scheme/host/port.  Doesn't it follow that a URI with no path wants the default?

I had hope that this wasn't a change in behavior.

Reply:
Before, if you sent in a path, we would ignore it. Now, we don't ignore it any more.

Reply:
Derp ok.  Well I am not sure why that was ever desirable...so we can capture this in the release notes?

Reply:
I think that's totally fine. Do you want to pre-populate the changelog in the coming soon section so we don't forget?

Reply:
Sure...I'll do that.  And I'm going to bump to 0.5.0-SNAPSHOT since I don't think a minor .1 covers it.

Reply:
ready for another look.